empress/support_files/templates/side-panel.html

```diff
@@ -131,18 +131,49 @@
   </div>
   <hr />
   <p class="side-panel-inner-header">
-    Layout options
-  </p>
-  <p>
-    <label for="ignore-lengths-chk">Ignore branch lengths?</label>
-    <input id="ignore-lengths-chk" type="checkbox" class="empress-input">
-  </p>
-  <p class="side-panel-notes indented">
-    If checked, branch lengths are ignored when drawing the tree. Note that the
-    root node is never drawn with any length, regardless of if this
-    is checked or not.
+    Branch lengths
   </p>
+  <div id="branch-length-warning">
+    <p class="side-panel-notes">
+      Warning: the "Ignore branch lengths" and "Make ultrametric" options display
+      modified branch lengths that can be useful for aesthetics. The resulting
+      trees should not be interpreted with respect to their branch lengths.
+    </p>
+  </div>
+  <div id="branch-length-method-container">
+    <p>
+      <label for="normal-lengths-chk">Normal</label>
+      <input type="radio" id="normal-lengths-chk" name="branches-radio" value="normal"
+             class="empress-input" checked="true">
+    </p>
+    <p class="side-panel-notes indented">
+      The branch lengths shown correspond to the branch lengths in the original tree.
+    </p>
+    <p>
+      <label for="ignore-lengths-chk">Ignore branch lengths</label>
+      <input type="radio" id="ignore-lengths-chk" name="branches-radio" value="ignore"
+             class="empress-input">
+    </p>
+    <p class="side-panel-notes indented">
+      If checked, branch lengths are ignored when drawing the tree. Note that the
+      root node is never drawn with any length, regardless of if this
+      is checked or not.
```

Comment:
```suggestion
      If checked, all branches are drawn with the same length. Note that the
      root node is never drawn with any length, regardless of if this
      is checked or not.
```

Feel free to change the wording -- I realized just now that saying this "ignores" branch lengths might be slightly misleading, since technically the make ultrametric setting also ignores the branch lengths in the sense that it modifies them. But this is extremely minor so not a big deal